pyproject.toml

```diff
@@ -105,6 +105,7 @@ py = "1.4.33"
 python3-openid = "3.1.0"
 requests-oauthlib = "0.8.0"
 webencodings = "0.5.1"
+redo = "^2.0.3"
```

Comment:
Did I do it right @callahad ?

I used `docker-compose exec web poetry add redo`

Reply:
Yep, that looks totally reasonable on the Poetry side

Reply:
Thanks. I'll wait for @escattone to review and merge.